lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -252,7 +251,10 @@ class CategoryRenderer {
 
       // Push grouped audits as a group.
       const groupDef = groupDefinitions[groupId];
-      const auditGroupElem = this.renderAuditGroup(groupDef, opts);
+      // Expanded by default!
+      // 1. The 'failed' clump has all groups expanded.
+      // 2. If a clump is collapsed (passed, n/a), we want the groups within to already be expanded
+      const auditGroupElem = this.renderAuditGroup(groupDef, {expandable: false});
```

Comment:
this is a little confusing, it's probably because I don't remember what our opts actually do at this point 😆 so `expandable` means `isExpansionToggleable` and it's expanded by default?

Reply:
agree it's a funny name. `expandable` means `collapsedAndCanBeExpanded`